examples/browserify-gulp-example/gulp/tasks/browserify.js

```diff
@@ -49,7 +49,7 @@ gulp.task('browserify', function(callback) {
         .on('end', reportFinished);
     };
 
-    bundler.transform(babelify.configure());
+    bundler.transform("babelify", {presets: ["es2015", "react"]});
```

Comment:
The `presets` are already defined in `.babelrc`, so I think you would only need to remove the `.configure()`.